src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssGetAcl.java

```diff
@@ -9,5 +9,127 @@
  */
 package zowe.client.sdk.zosfiles.uss.methods;
 
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import zowe.client.sdk.core.ZosConnection;
+import zowe.client.sdk.rest.PutJsonZosmfRequest;
+import zowe.client.sdk.rest.Response;
+import zowe.client.sdk.rest.ZosmfRequest;
+import zowe.client.sdk.rest.ZosmfRequestFactory;
+import zowe.client.sdk.rest.type.ZosmfRequestType;
+import zowe.client.sdk.utility.EncodeUtils;
+import zowe.client.sdk.utility.FileUtils;
+import zowe.client.sdk.utility.ValidateUtils;
+import zowe.client.sdk.zosfiles.ZosFilesConstants;
+import zowe.client.sdk.zosfiles.uss.input.GetAclParams;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Provides Unix System Services (USS) getfacl functionality
+ * <p>
+ * <a href="https://www.ibm.com/docs/en/zos/2.4.0?topic=interface-zos-unix-file-utilities">z/OSMF REST API</a>
+ *
+ * @author James Kostrewski
+ * @version 2.0
+ */
 public class UssGetAcl {
+
+        private final ZosConnection connection;
+        private ZosmfRequest request;
+
+        /**
+        * UssGetAcl Constructor
+        *
+        * @param connection connection information, see ZosConnection object
+        */
+        public UssGetAcl(ZosConnection connection) {
+            ValidateUtils.checkConnection(connection);
+            this.connection = connection;
+        }
+
+        /**
+        * Alternative UssGetAcl constructor with ZoweRequest object. This is mainly used for internal code unit testing
+        * with mockito, and it is not recommended to be used by the larger community.
+        *
+        * @param connection connection information, see ZosConnection object
+        * @param request any compatible ZoweRequest Interface object
+        * @throws Exception processing error
+        */
+        public UssGetAcl(ZosConnection connection, ZosmfRequest request) throws Exception {
+            ValidateUtils.checkConnection(connection);
+            ValidateUtils.checkNullParameter(request == null, "request is null");
+            this.connection = connection;
+            if (!(request instanceof PutJsonZosmfRequest)) {
+                throw new Exception("PUT_JSON request type required");
+            }
+            this.request = request;
+        }
+
+        public String getAclWithCommas(String fileNamePath) throws Exception {
```

Comment:
At this time I think this method's existent and naming is awkward. Let's remove it and add a boolean value to the main getAcl method second parameter to specify to receive the output with commas or not and handle the logic there.